src/nikonmn_int.hpp

```diff
@@ -124,7 +124,9 @@ namespace Exiv2 {
         //! Return read-only list of built-in Auto Focus tags
         static const TagInfo* tagListAf();
         //! Return read-only list of built-in Auto Focus 2 tags
-        static const TagInfo* tagListAf2();
+        static const TagInfo* tagListAf21();
+        //! Return read-only list of built-in Auto Focus 2 tags
+        static const TagInfo* tagListAf22();
```

Comment:
Both `tagListAf21()` and `tagListAf22()` has exactly the same documentation. Should we make clear the difference between them here?

Reply:
I've changed the comments to clarify the difference.

Reply:
They have the same documentation.